.circleci/config.yml

```diff
@@ -0,0 +1,36 @@
+version: 2
+jobs:
+  build:
+    working_directory: ~/repo
+    docker:
+      - image: circleci/node:lts
+    steps:
+      - checkout:
+      - restore_cache:
+          keys:
+            - v1-dependencies-{{ checksum "package.json" }}
+            - v1-dependencies-
+      - run:
+          name: update
+          command: 'sudo apt-get update -y && sudo apt-get -y install rsync'
```

Comment:
Because if I dont then [this](https://github.com/BoltzExchange/boltz-middleware/blob/51a9d4c51da3939fb21fa5571ed1a94a53b89ba7/package.json#L23) wont work.